### PR TITLE
Add avg-antivirus :latest

### PR DESCRIPTION
**Pull request by ninotosh**
This will install the latest AVG Antivirus for Mac.
The application offers “Uninstall AVG Antivirus…” in the menu, but I couldn’t figure out how to put an uninstall stanza :sweat:

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) geared for that.

File: Casks/avg-antivirus.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'avg-antivirus' do
+  version :latest
+  sha256 :no_check
+
+  # avg.mac.avcdn.net/macosx was verified as official when first introduced to the cask
+  url 'http://avg.mac.avcdn.net/macosx/avg/avg_antivirus_for_mac_online.dmg'
```
Comment (vitorgalvao) [inline Casks/avg-antivirus.rb]: Where did you get this URL?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/avg-antivirus.rb (hunk 1 above)
Comment (ninotoshi) [inline Casks/avg-antivirus.rb]: The green button on http://www.avg.co.jp/avg-antivirus-for-mac redirects there.

Comment (vitorgalvao) [inline Casks/avg-antivirus.rb]: > The green button on http://www.avg.co.jp/avg-antivirus-for-mac redirects there.

But not the one on the homepage you linked to. That one redirects to cnet.

Comment (ninotoshi) [inline Casks/avg-antivirus.rb]: I assume there is one universal binary, and `homepage` is only a reference.

Comment (vitorgalvao) [inline Casks/avg-antivirus.rb]: Confirmed they are the same. Still needs `uninstall` before it can be accepted, though.

Comment (miccal) [inline Casks/avg-antivirus.rb]: `uninstall` added.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**